### PR TITLE
Tiny clean up.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: optimizer/optimizer.go [hunk 1]
```diff
@@ -39,16 +39,18 @@ func Optimize(ctx context.Context, node ast.Node) (plan.Plan, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	alts, err := plan.Alternatives(p)
+	err = plan.Refine(p)
```
Comment (coocood) [inline optimizer/optimizer.go]: `Refine` should be called after `Alternatives`

Comment (qiuyesuifeng) [inline optimizer/optimizer.go]: Got it, my fault.

**Comment by qiuyesuifeng** [comment]
@coocood  PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**